docs/databags.md

```diff
@@ -33,6 +33,15 @@ The License Hash is part of a data bag item encrypted with [Chef Vault](https://
 
 * `[A Decriptive File-Name]` - Corresponding License (XML) contents. (There can be many of these) Remember to change newlines to `\n` to conform to proper JSON format.
 
+License Pool Hash
+------------
+This hash is used to define multiple license pools. Indexers can be assigned to a specific pool and license quota can be set for each pool by configuring this databag. 
```

Comment:
similar to the other sections here, this should say it's part of a plain text data bag item.